entrypoint.sh

```diff
@@ -2,9 +2,18 @@
 
 if [[ -n "$BASE_URL" ]]; then
     # Run sed command for all files in the static directory
-    sed -i 's|"/_next/static|"'$BASE_URL'/_next/static|g' static/*.*
-    sed -i 's|"/admin/_next/static|"'$BASE_URL'/admin/_next/static|g' static/admin/*.*
-    sed -i 's|"/api/|"'$BASE_URL'/api/|g' static/*.*
+    # Replace this: url(/_next with this: url($BASE_URL/_next (Mostly for link in CSS files)
+    find static -type f -name "*.*" -exec sed -i 's|url(/_next|url('"$BASE_URL"'/_next|g' {} +
```

Comment:
can't we simplify this sed expression, if we use as asset dir in NextJs config a distinct prefix like: `https://static-assets.ragapp.org/` and replace using sed for this string?

Reply:
I'm not sure about that because the static links are constructed in many way. Let me have a test to see if we can simplify that.